components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -10,291 +10,788 @@ toc = true
     weight = 30
 +++
 
-This guide shows you how to manage Chef Automate IAM v2 policies and roles from the command line.
-Any command with a browser equivalent includes brief directions for locating it on the corresponding page.
-See the [IAM v2 Overview]({{< relref "iam-v2-overview.md" >}}) for an overview and exposition of new concepts.
-See the [IAM v2 User's Guide]({{< relref "iam-v2-guide.md" >}}) for more information on setting up and using IAM v2 features.
+This API reference details Chef Automate IAM v2 features from the command line.
+If you are not already on IAM v2, [upgrade to IAM v2]({{< relref "iam-v2-guide.md#upgrade-to-iam-v2" >}}) first.
```

Comment:
Should we provide more context before just telling them to upgrade?

Reply:
@tylercloke what kind of context should we add?